tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "module": "commonjs",
     "esModuleInterop": true,
     "target": "es6",
+    "lib": ["es2021", "dom"],
```

Comment:
We shouldn't need dom here, but the AWS dependency fails if we don't have it, because the type interface uses ReadableStream